jib-core/src/test/java/com/google/cloud/tools/jib/http/AuthorizationTest.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.jib.http;
+
+import com.google.common.collect.Multimap;
+import org.junit.Assert;
+import org.junit.Test;
+
+/** Tests for {@link Authorization}. */
+public class AuthorizationTest {
+  @Test
+  public void testDecode_dockerToken() {
```

Comment:
Instructions to help generate these encoded strings if we need to do it again?

Reply:
trial and error, of course.

Reply:
Is there a way to easily synthesize these encoded values from human-readable JSONs? Can you document that as comments in the file?